src/v0/util/facebookUtils/networkHandler.js

```diff
@@ -100,12 +100,16 @@ const errorDetailsMap = {
   190: {
     460: new ErrorDetailsExtractorBuilder()
       .setStatus(400)
+      .setStat('accessTokenExpired')
       .setMessage(
         'The session has been invalidated because the user changed their password or Facebook has changed the session for security reasons',
       )
       .build(),
     default: new ErrorDetailsExtractorBuilder()
       .setStatus(400)
+      .setStat({
+        [tags.TAG_NAMES.ERROR_TYPE]: tags.ERROR_TYPES.ACCESS_TOKEN_EXPIRED,
+      })
```

Comment:
```suggestion
      .setStat({
        [tags.TAG_NAMES.META]: tags.ERROR_TYPES.UNAUTHORISED,
      })
```

Please check spelling and make necessary changes